src/lib/fetch-retry.js

```diff
@@ -0,0 +1,23 @@
+function wait(delay) {
+  return new Promise((resolve) => setTimeout(resolve, delay));
+}
+
+export function fetchRetry(url, options = {}, retries) {
```

Comment:
What do you think of giving the `retries` arg a default value of 1?
```suggestion
export function fetchRetry(url, options = {}, retries = 1) {
```